plugin-mainmenu/lxqtmainmenu.cpp

```diff
@@ -253,7 +253,7 @@ void LXQtMainMenu::settingsChanged()
     mSearchEdit->setText(QString{});
     mFilterMenu = settings()->value(QStringLiteral("filterMenu"), true).toBool();
     mFilterShow = settings()->value(QStringLiteral("filterShow"), true).toBool();
-    mFilterClear = settings()->value(QStringLiteral("filterClear"), false).toBool();
+    mFilterClear = settings()->value(QStringLiteral("filterClear"), true).toBool();
```

Comment:
For the sake of correctness, you need to change the default value here -> https://github.com/lxqt/lxqt-panel/blob/513a7fd45f5adbc970db8bb9078cec923e5e3252/plugin-mainmenu/lxqtmainmenuconfiguration.cpp#L146 also.

Reply:
Done, thanks for spotting that.